### DISCUSSION TITLE
"Cannot find type 'AnyHashableSendable' in scope" bug on build on 1.0.2

### DISCUSSION THREAD

**Discussion by ckse93** [pointfreeco/swift-sharing#12] [General]
### Description

<img width="1512" alt="Screenshot 2024-12-02 at 1 09 23 PM" src="https://github.com/user-attachments/assets/87a47184-6e12-4f49-a36b-5b88711e5b97">

experiencing "Cannot find type 'AnyHashableSendable' in scope" on build 
using swift version 6
using composable architecture 1.12.1

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-sharing/issues) or [discussion](https://github.com/pointfreeco/swift-sharing/discussions).

### Sharing version information

1.0.2

### Destination operating system

Vision OS 2.0

### Xcode version information

Version 16.0 (16A242d)

### Swift Compiler version information

```shell
Apple Swift version 6.0 (swiftlang-6.0.0.9.10 clang-1600.0.26.2)
```

**Comment by mbrandonw** [comment]
Hi @ckse93, you cannot use an old version of TCA _and_ this library. You will need to wait until the PR I mentioned previously is merged and released.

But also, there is no reason for you to use this library with TCA 1.12. You can just keep using 1.12 alone since it already has all of the state sharing tools in it.

Since this isn't an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!
